contrast_api/data_migration_functionality/create_study.py

```diff
@@ -21,7 +21,7 @@ def create_study(item: dict, unconsciousness):
         study_type = StudyTypeChoices.UNCONSCIOUSNESS
         funding = ""
         affiliations = ""
-        abbreviated_source_title = ""
+        abbreviated_source_title = item["Journal"]
```

Comment:
btw, for the next time we do this, I'd consider using a dataclass instead of a typeless item